### DISCUSSION TITLE
Changing content redirection

### DISCUSSION THREAD

**Discussion by jvegh** [community/community#143634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I would like to ask for your help.
I do have a developer site
https://github.com/jvegh/AbstractNeuronDynamics/
and I wanted to attach a public documentation to it.
https://jvegh.github.io/AbstractNeuronDynamics/

Originally, I did make a site in its index.html, with part-of documentation, temporarily,

        <meta http-equiv="refresh" content="1;url=docs/index.html">
	https://github.com/jvegh/AbstractNeuronDynamics/blob/main/docs/index.html
and it worked fine. Now I am nearer to a final documentation, and the former one will be referred to
from the new one, so I uploaded the new content to a different directory

        <meta http-equiv="refresh" content="1;url=www/manual/index.html">
	https://github.com/jvegh/AbstractNeuronDynamics/blob/main/www/manual/index.html

I have updated the redirection correspondingly, and it works fine on my local computer.
I see the correct content in the index file on your server. Still, the old (and now invalid) 
redirection takes place. 
Maybe I need to refresh/delete some cache?

**Comment by Galcedion** [comment]
Just read around a bit, since it sounds like a browser problem. Although I can't say with certainty that refresh is treated as 301 redirect, it might be interpreted as such by your browser, being permanently cached (Browsers tend to behave differently local vs internet). For Firefox and probably other Gecko browsers you can check the cache via `about:cache` and manage the website data via `about:preferences#privacy` under the section *Cookies and Site Data*. For Chrome and the like I found this [Google Chrome Help](https://support.google.com/chrome/thread/16531954/clear-cache-for-specific-website-in-google-chrome?hl=en).
You might want to add something like `<meta http-equiv="cache-control" content="no-cache">` to avoid such issues in the future or move the redirect to an .htaccess or the webserver in general.

**Comment by M-tech-cmd** [comment]
1. Clear Browser Cache: Clear your browser’s cache to ensure it’s not using the old cached version of your site. You can usually do this from the browser settings under privacy or history options.

2. Force Refresh: Perform a hard refresh on your browser by pressing Ctrl + F5 (Windows) or Cmd + Shift + R (Mac) to bypass the cache.

3.GitHub Pages Cache: Sometimes, GitHub Pages itself might cache the old redirection. You can try the following:

Rename the index.html file: Temporarily rename your index.html file to something else, push the changes, and then rename it back to index.html and push again. This can sometimes force GitHub Pages to update the cache.

Add Cache-Control Headers: Add cache-control headers to your HTML files to control caching behavior. For example, you can add the following meta tag to your HTML files:
HTML

<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
<meta http-equiv="Pragma" content="no-cache">
<meta http-equiv="Expires" content="0">

Check GitHub Pages Settings: Ensure that your GitHub Pages settings are correctly pointing to the right branch and folder.

Wait for Cache Expiry: Sometimes, it might just take a bit of time for the cache to expire and the new redirection to take effect.
